website/docs/contributing/writing-guide.md

```diff
@@ -50,11 +50,24 @@ Examples:
 
 To link to a file in the same directory, use the file name.
 
-- `[file in the same directory](writing-guide.md)`
+```text
+[file in the same directory](writing-guide.md)
+```
 
 To link to file in a different directory, specify the relative file path.
 
-- `[file in a different directory](../basic-operations/read.md)`.
+```text
+[file in a different directory](../basic-operations/read.md)
+```
+
+When referencing files such as configuration files, specs, or internal definitions on GitHub, ensure that the link is to a release branch, not the `main` branch.
+This is important because the `main` branch may change frequently, and links to it may break.
+
+For example, if you want to link to the FerretDB Data API OpenAPI 3.0 specification, use the following link:
+
+```text
+[FerretDB Data API OpenAPI 3.0 specification](https://raw.githubusercontent.com/FerretDB/FerretDB/refs/tags/v2.2.0/internal/dataapi/api/openapi.json)
```

Comment:
Interesting how this example contradicts the text

Reply:
The contradiction here is the tag vs branch, right?